code/__DEFINES/access.dm

```diff
@@ -150,3 +150,207 @@
+#define ACCESS_FLAG_AWAY				(1 << 6)
+/// Displayed name for Special accesses that ordinaryily shouldn't be on ID cards.
+#define ACCESS_FLAG_SPECIAL_NAME		"Special"
+/// Bitflag for Special accesses that ordinaryily shouldn't be on ID cards. See CULT_ACCESS.
+#define ACCESS_FLAG_SPECIAL				(1 << 7)
+
+/// This wildcraft flag accepts any access level.
+#define WILDCARD_FLAG_ALL			ALL
+/// Name associated with the all wildcard bitflag.
+#define WILDCARD_NAME_ALL			"All"
+/// Access flags that can be applied to common wildcard slots.
+#define WILDCARD_FLAG_COMMON 		ACCESS_FLAG_COMMON
+/// Name associated with the common wildcard bitflag.
+#define WILDCARD_NAME_COMMON		ACCESS_FLAG_COMMON_NAME
+/// Access flags that can be applied to command wildcard slots.
+#define WILDCARD_FLAG_COMMAND		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND
+/// Name associated with the command wildcard bitflag.
+#define WILDCARD_NAME_COMMAND		ACCESS_FLAG_COMMAND_NAME
+/// Access flags that can be applied to private command wildcard slots.
+#define WILDCARD_FLAG_PRV_COMMAND	ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND
+/// Name associated with the private command wildcard bitflag.
+#define WILDCARD_NAME_PRV_COMMAND	ACCESS_FLAG_PRV_COMMAND_NAME
+/// Access flags that can be applied to captain wildcard slots.
+#define WILDCARD_FLAG_CAPTAIN		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN
+/// Name associated with the captain wildcard bitflag.
+#define WILDCARD_NAME_CAPTAIN		ACCESS_FLAG_CAPTAIN_NAME
+/// Access flags that can be applied to centcom wildcard slots.
+#define WILDCARD_FLAG_CENTCOM		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN | ACCESS_FLAG_CENTCOM
+/// Name associated with the centcom wildcard bitflag.
+#define WILDCARD_NAME_CENTCOM		ACCESS_FLAG_CENTCOM_NAME
+/// Access flags that can be applied to syndicate wildcard slots.
+#define WILDCARD_FLAG_SYNDICATE		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN | ACCESS_FLAG_SYNDICATE
+/// Name associated with the syndicate wildcard bitflag.
+#define WILDCARD_NAME_SYNDICATE		ACCESS_FLAG_SYNDICATE_NAME
+/// Access flags that can be applied to offstation wildcard slots.
+#define WILDCARD_FLAG_AWAY			ACCESS_FLAG_AWAY
+/// Name associated with the offstation wildcard bitflag.
+#define WILDCARD_NAME_AWAY			ACCESS_FLAG_AWAY_NAME
+/// Access flags that can be applied to super special weird wildcard slots.
+#define WILDCARD_FLAG_SPECIAL		ACCESS_FLAG_SPECIAL
+/// Name associated with the super special weird wildcard bitflag.
+#define WILDCARD_NAME_SPECIAL		ACCESS_FLAG_SPECIAL_NAME
+/// Access flag that indicates a wildcard was forced onto an ID card.
+#define WILDCARD_FLAG_FORCED		ALL
+/// Name associated with the wildcard bitflag that covers wildcards that have been forced onto an ID card that could not accept them.
+#define WILDCARD_NAME_FORCED		"Hacked"
```

Comment:
```suggestion
#define ACCESS_FLAG_COMMON_NAME "Common"
/// Bitflag for Common ID card accesses. See COMMON_ACCESS.
#define ACCESS_FLAG_COMMON (1 << 0)
/// Displayed name for Command ID card accesses.
#define ACCESS_FLAG_COMMAND_NAME "Command"
/// Bitflag for Command ID card accesses. See COMMAND_ACCESS.
#define ACCESS_FLAG_COMMAND (1 << 1)
/// Displayed name for Private Command ID card accesses.
#define ACCESS_FLAG_PRV_COMMAND_NAME "Private Command"
/// Bitflag for Private Command ID card accesses. See PRIVATE_COMMAND_ACCESS.
#define ACCESS_FLAG_PRV_COMMAND (1 << 2)
/// Displayed name for Captain ID card accesses.
#define ACCESS_FLAG_CAPTAIN_NAME "Captain"
/// Bitflag for Captain ID card accesses. See CAPTAIN_ACCESS.
#define ACCESS_FLAG_CAPTAIN (1 << 3)
/// Displayed name for Centcom ID card accesses.
#define ACCESS_FLAG_CENTCOM_NAME "Centcom"
/// Bitflag for Centcom ID card accesses. See CENTCOM_ACCESS.
#define ACCESS_FLAG_CENTCOM (1 << 4)
/// Displayed name for Syndicate ID card accesses.
#define ACCESS_FLAG_SYNDICATE_NAME "Syndicate"
/// Bitflag for Syndicate ID card accesses. See SYNDICATE_ACCESS.
#define ACCESS_FLAG_SYNDICATE (1 << 5)
/// Displayed name for Offstation/Ruin/Away Mission ID card accesses.
#define ACCESS_FLAG_AWAY_NAME "Away"
/// Bitflag for Offstation/Ruin/Away Mission ID card accesses. See AWAY_ACCESS.
#define ACCESS_FLAG_AWAY (1 << 6)
/// Displayed name for Special accesses that ordinaryily shouldn't be on ID cards.
#define ACCESS_FLAG_SPECIAL_NAME "Special"
/// Bitflag for Special accesses that ordinaryily shouldn't be on ID cards. See CULT_ACCESS.
#define ACCESS_FLAG_SPECIAL (1 << 7)

/// This wildcraft flag accepts any access level.
#define WILDCARD_FLAG_ALL ALL
/// Name associated with the all wildcard bitflag.
#define WILDCARD_NAME_ALL "All"
/// Access flags that can be applied to common wildcard slots.
#define WILDCARD_FLAG_COMMON ACCESS_FLAG_COMMON
/// Name associated with the common wildcard bitflag.
#define WILDCARD_NAME_COMMON ACCESS_FLAG_COMMON_NAME
/// Access flags that can be applied to command wildcard slots.
#define WILDCARD_FLAG_COMMAND ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND
/// Name associated with the command wildcard bitflag.
#define WILDCARD_NAME_COMMAND ACCESS_FLAG_COMMAND_NAME
/// Access flags that can be applied to private command wildcard slots.
#define WILDCARD_FLAG_PRV_COMMAND ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND
/// Name associated with the private command wildcard bitflag.
#define WILDCARD_NAME_PRV_COMMAND ACCESS_FLAG_PRV_COMMAND_NAME
/// Access flags that can be applied to captain wildcard slots.
#define WILDCARD_FLAG_CAPTAIN ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN
/// Name associated with the captain wildcard bitflag.
#define WILDCARD_NAME_CAPTAIN ACCESS_FLAG_CAPTAIN_NAME
/// Access flags that can be applied to centcom wildcard slots.
#define WILDCARD_FLAG_CENTCOM ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN | ACCESS_FLAG_CENTCOM
/// Name associated with the centcom wildcard bitflag.
#define WILDCARD_NAME_CENTCOM ACCESS_FLAG_CENTCOM_NAME
/// Access flags that can be applied to syndicate wildcard slots.
#define WILDCARD_FLAG_SYNDICATE ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN | ACCESS_FLAG_SYNDICATE
/// Name associated with the syndicate wildcard bitflag.
#define WILDCARD_NAME_SYNDICATE ACCESS_FLAG_SYNDICATE_NAME
/// Access flags that can be applied to offstation wildcard slots.
#define WILDCARD_FLAG_AWAY ACCESS_FLAG_AWAY
/// Name associated with the offstation wildcard bitflag.
#define WILDCARD_NAME_AWAY ACCESS_FLAG_AWAY_NAME
/// Access flags that can be applied to super special weird wildcard slots.
#define WILDCARD_FLAG_SPECIAL ACCESS_FLAG_SPECIAL
/// Name associated with the super special weird wildcard bitflag.
#define WILDCARD_NAME_SPECIAL ACCESS_FLAG_SPECIAL_NAME
/// Access flag that indicates a wildcard was forced onto an ID card.
#define WILDCARD_FLAG_FORCED ALL
/// Name associated with the wildcard bitflag that covers wildcards that have been forced onto an ID card that could not accept them.
#define WILDCARD_NAME_FORCED "Hacked"
```

I believe this was added to the contribution guidelines, right?

Reply:
Done.